src/abstractdatatable/io.jl

```diff
@@ -45,7 +45,7 @@ function printtable(io::IO,
             if !isnull(dt[j][i])
                 if ! (etypes[j] <: Real)
                     print(io, quotemark)
-                    x = isa(dt[i, j], Nullable) ? _unsafe_get(dt[i, j]) : dt[i, j]
+                    x = isa(dt[i, j], Nullable) ? NullableArrays.unsafe_get(dt[i, j]) : dt[i, j]
```

Comment:
Ah, I was working on 0.5 when I wrote this and couldn't find the Base.unsafe_get. Will use @compat(unsafeget)